client/docs/source/concepts.rst

```diff
@@ -4,11 +4,18 @@ Concepts
 Test Development Kit (TDK)
 --------------------------
 
-The deliverable of this project is called the Test Development Kit, or TDK for short.  It is like an SDK, and you can use it in the same way.  The only difference is that it is meant to be consumed by the testing library pytest, rather than a python program written by someone.  
+The deliverable of this project is called the Test Development Kit, or TDK for short.  It is like an SDK, and you can use it in the same way.  The only difference is that it is meant to be consumed by the testing library pytest, rather than a python program written by someone.  At the highest level it is divided into two phases:  *orchestration* and *execution*.  
 
-Architecture
-------------
+Orchestration
+-------------
 
-.. image:: images/architecture.png
+This is the phase in which resources are assembled according to what is needed in the execution phase.  Currently in the TDK there are two ways to do this, but the method of doing this is not restrained in any way.  The only requirement is that the orchestration phase output a compatible config.json file for the execution phase to use.
```

Comment:
I know this is picky, but this is a little hard to parse: there are no constraints... except that there only two ways to do it.   Maybe something like "TDK orchestration consists of creating a config.json file: any process that does that will work.  Currently, there are two such processes implemented"